### DISCUSSION TITLE
Video Downloads: Just Audio-Only or Video-Only?

### DISCUSSION THREAD

**Discussion by GitAlexToTheGreek** [FreeTubeApp/FreeTube#6167] [General]
Hi there 👋

![2024-11-14-008-](https://github.com/user-attachments/assets/4c5480aa-daf2-4ab8-91eb-2ab4e48e7e6e)

It seems like there's no option to download files including both audio and video?

Can anyone clarify if it's possible to tweak a setting in order to enable such?

Thank you

**Comment by absidue** [comment]
Please read https://github.com/FreeTubeApp/FreeTube/issues/5030#issuecomment-2080462240
